src/go/vtunnel/test/e2e/connectivity_test.go

```diff
@@ -41,8 +41,8 @@ import (
 )
 
 var (
-	wslTarballName = "distro-0.21.tar"
-	wslTarballURL  = "https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/releases/download/v0.21/distro-0.21.tar"
+	wslTarballName = "distro-0.27.tar"
+	wslTarballURL  = fmt.Sprintf("https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/releases/download/v0.27/%s", wslTarballName)
```

Comment:
Should these be `const` instead of `var`? I can't imagine these changing at runtime.

It's somewhat odd that there isn't a separate constant for the version, but that's probably fine.

Reply:
`fmt.Sprintf` cannot be a constant which means we have to break them up into two groups of const and vars, I think the way it is is fine. However, I like the idea of declaring the version number separately